### PR TITLE
Prime update

### PR DESCRIPTION
**Pull request by GCheung55**
Prime removed shell and switched to use moutjs.

Do likewise, but keep the array and object methods and tests.

Update to prime@0.4

### CODE REVIEW

**Comment by GCheung55** [comment]
@arian: I left mixin prime in the repo. Should it be removed since prime@0.4 includes mixin behavior?

File: array/contains.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var indexOf = require('prime/array/indexOf')
+var indexOf = require('mout/array/indexOf')
```
Comment (arian) [inline array/contains.js]: http://moutjs.com/docs/latest/collection.html#contains

**Comment by arian** [comment]
Functions that are in mout, or look very much what's in mout, can be removed entirely. Then prime-util can be tagged with a new minor version according to semver.

Tests for those functions can be removed as well. Maybe tests can separated over multiple files as well.

File: array/contains.js (hunk 1 above)
Comment (GCheung55) [inline array/contains.js]: The difference here is the signature. `mout/array/contains` doesn't let you define a starting index. I don't know if this matters? If not, I can remove it and the test associated to it.

**Pull request merged**